example/src/screens/CustomHtmlUi.tsx

```diff
@@ -1,26 +1,51 @@
-import React, { useCallback } from 'react';
+import React, { useCallback, useRef } from 'react';
 import { View, Platform, StyleSheet } from 'react-native';
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
 import { useFocusEffect } from '@react-navigation/native';
 import {
   Event,
   usePlayer,
   PlayerView,
   SourceType,
+  CustomMessageHandler,
 } from 'bitmovin-player-react-native';
 import { useTVGestures } from '../hooks';
+import Button from '../components/Button';
+import { RootStackParamsList } from '../App';
+
+type CustomHtmlUiProps = NativeStackScreenProps<
+  RootStackParamsList,
+  'CustomHtmlUi'
+>;
 
 function prettyPrint(header: string, obj: any) {
   console.log(header, JSON.stringify(obj, null, 2));
 }
 
-export default function BasicPlayback() {
+export default function CustomHtmlUi({ navigation }: CustomHtmlUiProps) {
   useTVGestures();
 
+  const customMessageHandler = useRef(
+    new CustomMessageHandler(
+      (message: string, payload: string | undefined) => {
```

Comment:
It would be very nice if we had named parameters here, what do you think about that? Outside of an IDE this is unreadable otherwise.